bindings/python/iota_sdk/types/balance.py

```diff
@@ -62,8 +62,14 @@ class NativeTokensBalance:
         metadata: Some metadata of the native token.
     """
     token_id: HexStr
-    total: HexStr
-    available: HexStr
+    total: int = field(metadata=config(
+        encoder=hex,
+        decoder=lambda v: int(v, 0)
+    ))
+    available: int = field(metadata=config(
+        encoder=hex,
+        decoder=lambda v: int(v, 0)
```

Comment:
There is no `hex` for decoder ?

Reply:
the decoder defaults to `int` because that's the type, but it cannot decode hex without the second argument (radix, which is 0 to indicate that python should figure that part out)